src/core/components/choice-card/styles.ts

```diff
@@ -29,6 +29,16 @@ export const flexContainer = css`
 	}
 `
 
+export const gridContainer = ({columns} : {columns : number}) => css`
+	width: 100%;
+
+	${from.mobileLandscape} {
+		display: grid;
+		grid-template-columns: repeat(${columns}, 1fr);
+		row-gap: ${space[2]}px;
+	}
+`
+
```

Comment:
To avoid dynamic styles, we could extract the dynamic part into a separate function that populates an object:

```tsx
export const gridContainer = css`
	width: 100%;
	${from.mobileLandscape} {
		display: grid;
		row-gap: ${space[2]}px;
	}
`

const gridColumnsStyle = (columns: Columns) = css`
  	${from.mobileLandscape} {
		grid-template-columns: repeat(${columns}, 1fr);
	}
`

export const gridColumns: {[key in Columns]: SerializedStyles } = {
  2: gridColumnsStyle(2),
  3: gridColumnsStyle(3),
  4: gridColumnsStyle(4),
  5: gridColumnsStyle(5),
}

```

For reference, we've employed this pattern [elsewhere](https://github.com/guardian/source/blob/main/src/core/components/layout/components/stack/styles.ts)